wallet/src/de/schildbach/wallet/ui/dashpay/work/BroadcastIdentityVerifyWorker.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 Dash Core Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Aren't we using MIT in the wallet?

Reply:
GPL3, but some how there are some Apache 2.0 licenses in the headers